pyproject.toml

```diff
@@ -38,7 +38,7 @@ metadata = false
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 alpineer = ">=0.1.9"
-mibi-bin-tools = "0.2.9"
+mibi-bin-tools = "0.2.10"
```

Comment:
Will this require any changes to the readme, releases, etc?

Reply:
We will need to make a new release so that users can have the updated `mibi-bin-tools==0.2.10`.

Reply:
No changes to the readme needed, however.